private/pkg/pluginrpcutil/pluginrpcutil.go

```diff
@@ -23,29 +23,3 @@ import (
 func NewRunner(delegate command.Runner, programName string, programArgs ...string) pluginrpc.Runner {
 	return newRunner(delegate, programName, programArgs...)
 }
-
-// RunnerProvider provides pluginrpc.Runners for program names and args.
-type RunnerProvider interface {
-	NewRunner(programName string, programArgs ...string) pluginrpc.Runner
-}
-
-// RunnerProviderFunc is a function that implements RunnerProvider.
-type RunnerProviderFunc func(programName string, programArgs ...string) pluginrpc.Runner
-
-// NewRunner implements RunnerProvider.
-func (r RunnerProviderFunc) NewRunner(programName string, programArgs ...string) pluginrpc.Runner {
-	return r(programName, programArgs...)
-}
-
-// NewRunnerProvider returns a new RunnerProvider for the command.Runner.
-func NewRunnerProvider(delegate command.Runner) RunnerProvider {
```

Comment:
To avoid import of `bufconfig` in a `pkg` dir this has been moved to `bufcheck`.